Modules/ThirdParty/ZLIB/src/itkzlib-ng/CMakeLists.txt

```diff
@@ -1307,3 +1307,6 @@ endif()
 
 include(CPack)
 endif() # ITK change
+# Begin ITK additions
+mark_as_advanced(WITH_GTEST ZLIBNG_ENABLE_TESTS)
+# End ITK additions
```

Comment:
Sorry, I have the same question here: would it be possible to move these ITK additions to the ITK specific CMakeLists.txt at "Modules/ThirdParty/ZLIB"?

Reply:
Thanks for addressing my comments @thewtex ! 👍